src/cloud/webhooks/userAfterSave.js

```diff
@@ -26,17 +28,37 @@ const userAfterSave = async request => {
     );
   }
 
-  // Create new user chat channels
+  // Get parse role
+  const onboarding_role = await new Parse.Query(Parse.Role).equalTo('name', 'ONBOARDING_ADMIN').first();
+  // Get twilio users
+  const users = await new Twilio().client.chat.services(process.env.TWILIO_SERVICE_SID).users.list();
+  // Filter users by the desired role
+  const onboardingAdmins = users.filter(user => user.roleSid === onboarding_role.get('twilioRoleSID'));
```

Comment:
Check first if `onboarding_role.get('twilioRoleSID')` is defined. Otherwise could lead unexpected errors.